legal_db/models.py

```diff
@@ -1,11 +1,71 @@
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
+from django_countries.fields import CountryField
 
 
-class License(models.Model):
-    code = models.CharField(max_length=12)
-    title = models.CharField(max_length=50)
-    version = models.CharField(max_length=25, blank=True, null=True)
+class LegalResource(models.Model):
+    license = models.CharField(max_length=25, blank=True, null=True)
+    license_version = models.CharField(max_length=25, blank=True, null=True)
+    is_ported = models.BooleanField(blank=True, null=True)
+    contributor_name = models.CharField(max_length=120)
+    contributor_email = models.EmailField()
+    contribution_credit = models.BooleanField()
+    link = models.URLField(max_length=2000)
+    country = CountryField()
+    summary = models.TextField(blank=True, null=True)
     notes = models.TextField(blank=True, null=True)
+    publish = models.BooleanField(default=False)
```

Comment:
I think this field name is a bit confusing too. Maybe we can call it status and have it be a ChoiceField instead of a boolean? we can have multiple statuses (unreviewed, review in progress, published, rejected), something like that.

I'll defer to @TimidRobot here too.

Reply:
Added as suggested.